test/test_list.py

```diff
@@ -396,12 +427,19 @@ def test_list_cat_mappings(capsys):
  'type': 'mapping',
  'uploaded_as': 'hst_cos_deadtab_0250.rmap',
  'useafter_date': '2050-01-01 00:00:00'}"""
-    assert out_to_check in out
+    default_shared_state.cleanup()
+    assert out_to_check1 in out
+    assert out_to_check2 in out
+    assert out_to_check3 in out
+    assert out_to_check4 in out
 
 
-def test_list_status(capsys):
+@mark.go
```

Comment:
was this intentional? same thing on L456

Reply:
Ah no, just leftover test marks. Cheers!